src/SampleLibraryTests/Setup.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Globalization;
+using System.IO;
+using System.Reflection;
+using NUnit.Framework;
+
+
+[SetUpFixture]
+public class Setup
+{
+    private string moduleRootFolder;
+    List<string> resolutionPaths;
+
+    [OneTimeSetUp]
+    public void RunBeforeAllTests()
```

Comment:
thanks for doing this - I do hope we can come up with something better long term though. I believe there is a task in the backlog for it.

One thing we explored in the past was pulling full dynamo builds directly from s3 - that worked pretty well. There was even a msbuild task for it.

Reply:
Having dynamo (runtime) installed on the test machine + 3dParty tests pointing to that install path sounds like a good approach .
Maybe the 3dparty dev should make sure to install dynamocore on the build machine (not sure if we should do it as part of the test project build process)
But if we do want to provide this convenience, we could also include a script as part of the test nuget, that the user could use to download a specific version of dynamo (runtime) and set the proper paths in the test config file

Reply:
This might be a good idea for our test framework too (in order to segregate test deps from the runtime), separate out all the tests in a new solution and all tests would have a resolver to point to the dynamo output folder

Reply:
We do install dynamo runtime as part of tests in other places (e.g. GD repos).